### PR TITLE
Create pybitmessage.profile

### PR DESCRIPTION
**Pull request by 1dnrr**
tested on fedora-28 with pybitmessage 0.6.3.2

### CODE REVIEW

File: etc-fixes/0.9.52/pybitmessage.profile [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Firejail profile for default
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/default.local
```
Comment (SkewedZeppelin) [inline etc-fixes/0.9.52/pybitmessage.profile]: s/default/pybymessage

File: etc-fixes/0.9.52/pybitmessage.profile [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# generic gui profile
```
Comment (SkewedZeppelin) [inline etc-fixes/0.9.52/pybitmessage.profile]: remove

File: etc-fixes/0.9.52/pybitmessage.profile [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+notv
+novideo
+protocol unix,inet,inet6,netlink
+seccomp
+shell none
+
+disable-mnt
+private-bin pybitmessage,python*,sh,ldconfig,env,bash,stat
+private-dev
+private-etc PyBitmessage,PyBitmessage.conf,Trolltech.conf,fonts,gtk-2.0,hosts,ld.so.cache,ld.so.preload,localtime,pki,resolv.conf,selinux,sni-qt.conf,system-fips,xdg
```
Comment (SkewedZeppelin) [inline etc-fixes/0.9.52/pybitmessage.profile]: add `ca-certificates,ssl,pki,crypto-policies`

File: etc-fixes/0.9.52/pybitmessage.profile [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+noblacklist /usr/sbin
+
+noexec /sbin
+noexec /usr/local/sbin
+noexec /usr/sbin
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
```
Comment (SkewedZeppelin) [inline etc-fixes/0.9.52/pybitmessage.profile]: add `include /etc/firejail/disable-interpreters.inc`

File: etc-fixes/0.9.52/pybitmessage.profile [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
```
Comment (SkewedZeppelin) [inline etc-fixes/0.9.52/pybitmessage.profile]: add `nclude /etc/firejail/whitelist-var-common.inc`

File: etc-fixes/0.9.52/pybitmessage.profile [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# generic gui profile
+# depending on your usage, you can enable some of the commands below:
+noblacklist /sbin
+noblacklist /usr/local/sbin
+noblacklist /usr/sbin
+
+noexec /sbin
```
Comment (SkewedZeppelin) [inline etc-fixes/0.9.52/pybitmessage.profile]: noexec should be moved to bottom

File: etc-fixes/0.9.52/pybitmessage.profile [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+# Persistent local customizations
+include /etc/firejail/default.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# generic gui profile
+# depending on your usage, you can enable some of the commands below:
+noblacklist /sbin
+noblacklist /usr/local/sbin
+noblacklist /usr/sbin
```
Comment (SkewedZeppelin) [inline etc-fixes/0.9.52/pybitmessage.profile]: add

```
# Allow python (blacklisted by disable-interpreters.inc)
noblacklist ${PATH}/python2*
noblacklist ${PATH}/python3*
noblacklist /usr/lib/python2*
noblacklist /usr/lib/python3*
```

**Comment by 1dnrr** [comment]
Thanks you all. I created new pull request #2092 in /etc, with the updated file.

**Pull request closed without merging**